en/rules.md

```diff
@@ -86,16 +86,16 @@ Its Strength cannot exceed 66.
 *Slider AA*: 1 or below.
 
 ##### Timewarp:
-As Previously Mentioned: your Timewarp speed must be at minimum 60% of 1.5x speed for Double Time*, and 60% of 1x speed for everything else.
+As Previously Mentioned: your Timewarp speed must be at minimum 60% of 1.5x speed for Double Time*, and 1x speed for everything else.
 
 For AQN, That's:
-66 **Without DT**, 100 **With DT**.
+100 **Without DT**, 100 **With DT**.
 
 For osu!rx, That's:
-0.6 **Without DT**, 1.0 **With DT**.
+1.0 **Without DT**, 1.0 **With DT**.
```

Comment:
merge these two to `1.0 **With(out) DT**` maybe?

Reply:
same goes for the ones above/below

Reply:
no? why would you do that

Reply:
saving bytes of space

Reply:
nah bro im not that stupid keep your ass quiet

Reply:
> nah bro im not that stupid keep your ass quiet

? its a 15 byte save.

Reply:
I used With & Without, not a byte save, but looks nicer.